src/anniversary/sf33rd/Source/Game/RAMCNT.c

```diff
@@ -0,0 +1,185 @@
+#include "sf33rd/Source/Game/RAMCNT.h"
+#include "common.h"
+#include "sf33rd/AcrSDK/ps2/flps2debug.h"
+#include "sf33rd/AcrSDK/ps2/foundaps2.h"
+#include "sf33rd/Source/Common/MemMan.h"
+#include "sf33rd/Source/Game/debug/Debug.h"
+#include "sf33rd/Source/Game/texgroup.h"
+
+#define ERR_STOP                                                                                                       \
+    do {                                                                                                               \
+    } while (1)
+
+void disp_ramcnt_free_area() {
+    if (Debug_w[0xA]) {
+        flPrintColor(-0x71U);
```

Comment:
Considering this func takes a color in the form ARGB, this should be written like this
```suggestion
        flPrintColor(0xFFFFFF8F);
```